### DISCUSSION TITLE
Error uploading photos to github profile.

### DISCUSSION THREAD

**Discussion by danielclough** [community/community#82954] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Error Msg:
`Something went really wrong and we can’t process that picture.`

I have reviewed the answers given in: 
Error uploading photos to github profile. [48077](https://github.com/orgs/community/discussions/48077)
Profile pic not changing? [69303](https://github.com/orgs/community/discussions/69303)

Those seem to be out of date, because I made my file size larger in order to test if it was too small and got an error saying that max image size is 1MB (both claim 10MB max).

On Linux, I tried Firefox, Chromium, and Brave.
On Windows, I tried Firefox.

I do not have anti-virus software running, and I am not going to setup a VPN just to upload a picture.

I tried both .png and .jpg files.

I tried removing my old image... so now I have none.

I get `201` from https://github.com/upload/policies/avatars
```json
{
    "upload_url": "https://uploads.github.com/avatars",
    "header": {
        "Accept": "application/vnd.github.assets+json; charset=utf-8",
        "GitHub-Remote-Auth": "XXX...XXX"
    },
    "asset": {
        "owner_type": "User",
        "owner_id": XXXXXXX,
        "size": 111595,
        "content_type": "image/jpeg"
    },
    "form": {
        "owner_type": "User",
        "owner_id": XXXXXXX,
        "size": 111595,
        "content_type": "image/jpeg"
    },
    "same_origin": true,
    "upload_authenticity_token": "XXX...XXX"
}
```

Error associated with the failure is:
```
attachment-upload.ts:114 
       POST https://uploads.github.com/avatars 404 (Not Found)
```

I get a response with:
```json
{
  "message":"Not found",
  "request_id":"XXXX:XXXX:XXXXXX:XXXXXX:XXXXXXXX"
}
```

**Comment by MEClouds** [comment]
I recommend checking out the official GitHub documentation on personalizing profiles. This guide covers the steps to customize your GitHub profile, including changing your profile picture. You can find it [here](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile).

**Comment by danielclough** [comment]
I tried again (10hrs later) and it worked just fine.
